### DISCUSSION TITLE
How to let it snow?

### DISCUSSION THREAD

**Discussion by iptizer** [squidfunk/mkdocs-material#4685] [Q&A] [answered]
Cold times approching... therefore it would be nice to let it snow on our documentation. Basically an overlay with snowflakes.

I checked a bit with [particles.js](https://particles.js.org/) ([docs see here](https://github.com/matteobruni/tsparticles/blob/main/presets/snow/README.md#vue-2x-and-3x)) and there have been particles... but no documentation anymore.

Wondering whether someone already has a working solution? Shouldn't be that big of a deal.

**Comment by squidfunk** [comment, marked as answer]
Maybe there will be a new hidden feature flag in the coming days... 😉 Who knows?

**Reply by squidfunk** [reply]
- #4696
